packages/dev/docs/pages/blog/building-a-combobox.mdx

```diff
@@ -0,0 +1,128 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import comboboxAccessibilityUrl from 'url:../assets/combobox-accessibility.mp4';
+import comboboxScrollingUrl from 'url:../assets/combobox-scrolling-safari.mp4';
+import comboboxVisualViewportUrl from 'url:../assets/combobox-visual-viewport.mp4';
+import comboboxExampleImageUrl from 'url:../assets/combobox-example.png';
+import screenreaderSpreadsheetImageUrl from 'url:../assets/screenreader-spreadsheet-combobox.png';
+
+import {BlogPostLayout, Video, Image} from '@react-spectrum/docs';
+export default BlogPostLayout;
+
+---
+keywords: [combobox, accessibility, mobile, react spectrum, react, spectrum, interactions, touch]
+description: In this post we take a deeper look into some of the challenges faced when building an accessible and mobile friendly ComboBox.
+date: 2021-01-08
+author: '[Daniel Lu](https://github.com/LFDanLu)'
+---
+
+# Building a ComboBox
+
+<Image src={comboboxExampleImageUrl} alt="Screenshot of example ComboBox, select a major" style={{maxWidth: 'min(50%, 350px)', display: 'block', margin: '20px auto'}} />
+
+The [React Spectrum ComboBox](../react-spectrum/ComboBox.html) is now officially out of alpha as of January 14! For those who may be unfamiliar with a combobox, it is an input field that has a popover menu associated with it. The popover menu contains
+a list of values within that a user may select as the value for the combobox and is often filtered to display possible matches to the user's current input text. Finally, there is a button accompanying the input field used
+for toggling the open state of the popover menu.
+
+While the above may seem rather straight forward, a combobox contains a significant amount of complexity. In this blog post, we'll be covering the challenges we encountered with regards to ComboBox's mobile experience and accessibility.
+
+## Mobile experience
+
+### Onscreen Keyboard
+
+Like many of our other components, ComboBox displays its popover menu items in a tray when rendered on a mobile device or a smaller screen. This allows for a better mobile experience
```

Comment:
```suggestion
Like many of our other components, ComboBox displays its options in a tray rather than a popover when rendered on a mobile device or a smaller screen. This allows for a better mobile experience
```